static/js/common/upload-addon.js

```diff
@@ -327,12 +327,28 @@
 
                     $("<strong>").text(message).appendTo(upload_results);
 
+                    // Specific messages for unlisted addons.
+                    var isListed = $('#id_is_listed').is(':checked'),
+                        isSideload = $('#id_is_sideload').is(':checked');
+                    if (!isListed) {
+                      if (isSideload) {
+                        $("<p>").text(gettext("Your submission, as a sideloaded add-on, will go through a manual review.")).appendTo(upload_results);
+                      } else {
+                        if (warnings === 0) {
+                          $("<p>").text(gettext("Your submission passed validation and will therefore be automatically signed.")).appendTo(upload_results);
```

Comment:
'therefore' isn't used very often and makes this read a little weird.  If we just drop that word it sounds fine, I think.